nixos/modules/system/boot/systemd-journald.nix

```
@@ -0,0 +1,143 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
```

```suggestion
{ config, lib, pkgs, ... }:

```
I would suggest to keep the style of the formatting like it was before. This is also nixpkgs-fmt conform. win win.

---

Done :+1: